styles.css

```diff
@@ -0,0 +1,73 @@
+body{
+background-image:url("https://assets.bitdegree.org/online-learning-platforms/storage/media/2018/08/what-is-a-web-developer.jpg");
+/* text-align:center; */
+margin:0 auto;
+font-size:20;
+font-family:sans-serif;
+}
+hr{
+border-color:black;
+margin-bottom:20;
+width:80%;
+}
+p{
+font-size:20px;
+}
+h1{
+margin-top:50;
+}
+h1,p{
+color:white;
+text-align:center;
+
+}
+form{
+color:white;
+margin-top:30;
+}
+fieldset{
+width:500px;
+margin:0 auto;
+background-color:rgba(0,0,80,0.9);
+}
+input{
+display:block;
+height:40px;
+border-radius:10px;
+border-color:white;
+}
+
+input[type="text"]{
+width:100%;
+}
+input[type="number"]{
+width:100%;
+}
+input[type="email"]{
+width:100%;
+}
+textarea{
+width:100%;
+height:100px;
+padding:10px;
+margin-bottom:30px;
+margin-top:10px;
+}
+input[type="radio"]{
+display:unset;
+vertical-align:middle;}
+select{
+display:block;
+}
+input[type="checkbox"]{
+display:unset;
+vertical-align:middle;
+}
+#submit{
+display:block;
+margin:0 auto;
+height:40px;
+font-size:20px;
+background-color:green;
+width:100%
+}
```

Comment:
please check linters error to avoid your code from being messy and so it will be easy to understand .

Reply:
Good attempt to point out linter errors in your review. 👌🏿  However, the error you are pointing out does not relate to the line of code you are referring to. It is always a best practice that your in-line comments refer to the line or block of codes that your review is about. 🎯 

- [ ] Kindly make sure that all in-line comments relate to the code being referred to.